pkg/restapi/kms/operation/operations_test.go

```diff
@@ -298,6 +311,147 @@ func TestSignHandler(t *testing.T) {
 	})
 }
 
+func TestVerifyHandler(t *testing.T) {
+	t.Run("Success", func(t *testing.T) {
+		req, err := http.NewRequest(http.MethodPost, "", bytes.NewBuffer([]byte(validVerifyReq)))
+		require.NoError(t, err)
+
+		req = mux.SetURLVars(req, map[string]string{
+			"keystoreID": testKeystoreID,
+			"keyID":      testKeyID,
+		})
+
+		kh, err := keyset.NewHandle(signature.ED25519KeyTemplate())
+		require.NoError(t, err)
+
+		provider := NewMockProvider()
+		provider.MockStorage.Store.Store[testKeystoreID] = keystoreBytes(t)
+		provider.MockKMS.GetKeyValue = kh
+
+		op := New(provider)
+		handler := getHandler(t, op, verifyEndpoint, http.MethodPost)
+
+		rr := httptest.NewRecorder()
+		handler.Handle().ServeHTTP(rr, req)
+
+		require.Equal(t, http.StatusOK, rr.Code)
+	})
+
+	t.Run("Success: invalid signature", func(t *testing.T) {
```

Comment:
Is an invalid signature considered a success? i.e. an expected result?

(it might be, but I just want to double check)

Reply:
It's something that I was thinking about but still not sure on the best option. Probably, both are fine, and it's just a matter of style that we are willing to choose. I will try to investigate our other projects for similar scenarios and come up with some solution in the next PR (I am already working on it, and it involves a bunch of refactoring, e.g., returning JSON instead of raw bytes in response).

Reply:
Gotcha

Reply:
I had a similar question pop into my head when I saw we're returning `http.StatusOK` when the signature fails *but the service worked as expected*. Quick search didn't show me an appropriate response code and I'm inclined to think `200 OK` is fine.